SHTSensor.cpp

```diff
@@ -153,6 +153,71 @@ class SHTC1Sensor : public SHTI2cSensor
     }
 };
 
+//
+// class SHT2xSensor (SHT20, SHT21, SHT25)
+//
+
+class SHT2xSensor : public SHTI2cSensor
+{
+public:
+  SHT2xSensor(TwoWire &wire)
+      // clock stretching disabled
+      : SHTI2cSensor(0x40,     // i2cAddress
+                     0xF3F5,   // i2cCommand Hi: T, Lo: RH
+                     66,       // duration
+                     -46.85,   // a (sht_t_poly1)
+                     175.72,   // b (sht_t_poly2)
+                     65536.0,  // c (sht_t_poly3)
```

Comment:
this should likely be 65535

Reply:
Do you really want me to change this? I would prefer the datasheet value to avoid "stupid" questions ( like mine ;-) )